bin/install_jupyter.sh

```diff
@@ -35,7 +35,9 @@ apt-add-repository --yes ppa:gcpp-kalxas/jupyter
 apt-get update
 
 # Install latest jupyter notebook
-apt-get install --assume-yes jupyter-notebook python-qtconsole \
+apt-get install --assume-yes \
+        jupyter-notebook jupyter-client jupyter-core jupyter-nbconvert \
+        python-qtconsole jupyter-qtconsole \
```

Comment:
pyqt is already present on the disk. I think that jupyter-qtconsole is not very large. 
Rather, general interoperability between all parts in this new setup is a larger concern. Testing will be needed.
Python3 exposure is new to Live11. The libraries for python3, such as matplotlib, need to be filled out.